main.rb

```diff
@@ -0,0 +1,131 @@
+require './app'
+
+def welcome
+  puts 'Welcome to the School Library App'
+end
+
+def create_person(library)
+  print 'Do you want to create a Student (1) or a teacher (2)? [Input the number]:'
+
+  input = gets.chomp.to_i
+
+  print 'Name: '
+  name = gets.chomp
+
+  print 'Age: '
+  age = gets.chomp.to_i
+
+  if input == 1
+    print 'Has parent permission? y/n :'
+    permission = gets.chomp
+
+    library.create_student(age, name, permission == 'y')
+  else
+    print 'Specialization:'
+    spec = gets.chomp
+
+    library.create_teacher(age, name, spec)
+  end
+
+  puts input == 1 ? 'Student created Successfuly!' : 'Teacher created Successfuly!'
+
+  user_choice(library)
+end
```

Comment:
- [ ] When I try to create a person, I get the following error kindly fix it and also make sure after fixing it the creating rental works properly
>![image](https://user-images.githubusercontent.com/53402600/233783384-609de405-07ad-403c-9434-27013f05fe21.png)